ibr/ibr-sdk/build/ibr-sdk.js

```diff
@@ -285,55 +267,31 @@ typeof define === 'function' && define.amd ? define(['exports'], factory) :
   /**
      * Converts decoded ibr structure data into three.js Line objects
      and add them to scene with visibility set to false.
-     * @param {Object} structure structures decoded from raw ibr data.
+     * @param {IBRObject} curStructure IBRObject generated from current
+     structure data.
      * @param {number} structureIndex overall index of the structure.
      * @return {Map.<String, List.<Object>>} objects Layer name and
      corresponding list of three.js Line objects.
      */
-  function renderLayer(structure, structureIndex) {
+  function renderLayer(curStructure, structureIndex) {
```

Comment:
nit: just name this as `structure` or `ibrObject`, there's no looping going on to an end user so current doesn't really add to the meaning and understanding

Reply:
done